core/src/main/java/com/github/stephenvinouze/materialnumberpickercore/MaterialNumberPicker.kt

```diff
@@ -35,37 +34,31 @@ class MaterialNumberPicker : NumberPicker {
             field = value
             divider?.colorFilter = PorterDuffColorFilter(separatorColor, PorterDuff.Mode.SRC_IN)
         }
-
     var textColor: Int = DEFAULT_TEXT_COLOR
         set(value) {
             field = value
             updateTextAttributes()
         }
-
     var textStyle: Int = DEFAULT_TEXT_STYLE
         set(value) {
             field = value
             updateTextAttributes()
         }
-
     var textSize: Int = DEFAULT_TEXT_SIZE
         set(value) {
             field = value
             updateTextAttributes()
         }
-
     var editable: Boolean = DEFAULT_EDITABLE
         set(value) {
             field = value
             descendantFocusability = if (value) ViewGroup.FOCUS_AFTER_DESCENDANTS else ViewGroup.FOCUS_BLOCK_DESCENDANTS
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/StephenVinouze/MaterialNumberPicker/blob/ae55545cc2574a05af41ab00efd15ced1c007f9e/core/src/main/java/com/github/stephenvinouze/materialnumberpickercore/MaterialNumberPicker.kt#L55"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>